BSP/Simulator_V2/Src/BSP_Fans.c

```diff
@@ -0,0 +1,65 @@
+#include "BSP_Fans.h"
+#include "Simulator.h"
+
+#define NUM_FANS 4
+
+static bool initialized = false;
+static uint32_t speed[NUM_FANS] = {0,0,0,0};
```

Comment:
Just for style/management, shouldn't the initialization of the speed happen in BSP_Fans_Init? So you can change the `#define` for the NUM_FANS without having to change anything else in the file.